srcpkgs/docker-engine/template

```diff
@@ -0,0 +1,60 @@
+# Template file for 'docker-engine'
+pkgname=docker-engine
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/docker"
+hostmakedepends="pkg-config tar"
+makedepends="libbtrfs-devel device-mapper-devel libseccomp-devel"
+depends="containerd runc iptables xz git"
+short_desc="Pack, ship and run any application as a lightweight container - engine"
+maintainer="PWA COLLECTIVE <mail@pwa.cya.cx>"
+license="Apache-2.0"
+homepage="http://www.docker.io"
+system_groups="docker"
+
+_moby_commit=363e9a88a11be517d9e8c65c998ff56f774eb4dc	# https://github.com/moby/moby/commits/v$version
+_libnetwork_commit=fa125a3512ee0f6187721c88582bf8c4378bd4d7
+
+distfiles="
+	https://github.com/moby/moby/archive/v${version}.tar.gz>moby-$version.tar.gz
+	https://github.com/docker/libnetwork/archive/$_libnetwork_commit.tar.gz>libnetwork-$_libnetwork_commit.tar.gz"
+
+checksum="bcf651d75e5c80421e8cd3b0d47f3425e01047cf67aef0eda83b68776905a583
+ 9ce0fa64b00d9ef86971bdeb01cb24b02e6751c0d159ccb057521a41762750f4"
```

Comment:
`distfiles` and `checksum` should be right after `homepage` to follow the standard ordering, and use standard single-space indentation. Also, might as well put the moby url on the same line as `distfiles`, since that line will still be shorter than the libnetwork line anyway.